src/models.py

```diff
@@ -35,3 +41,142 @@ def __init__(self, hopeful):
 
     def __repr__(self):
         return "<Hopeful %i>" % self.hash_code
+
+class UserSchool(db.Model):
+    __tablename__ = 'userschool'
```

Comment:
nit: in general am vazut ca numele de tabele se scriu in general la plural; asa am folosit si in cazul `users` si `hopefuls`.